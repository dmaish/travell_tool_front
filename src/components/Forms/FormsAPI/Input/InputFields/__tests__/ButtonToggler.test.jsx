import React from 'react';
import ButtonToggler from '../ButtonToggler';

describe('<ButtonToggler />', () => {
  const choices = ['male', 'female'];

  it('renders as expected', () => {
    const wrapper = shallow(<ButtonToggler choices={choices} />);
    expect(wrapper).toMatchSnapshot();
  });
});
