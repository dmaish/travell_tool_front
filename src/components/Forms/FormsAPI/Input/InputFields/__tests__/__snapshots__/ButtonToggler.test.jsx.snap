// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonToggler /> renders as expected 1`] = `
<div
  className="input "
>
  <button
    className="bg-btn bg-btn--inactive"
    data-value="male"
    key="male"
    onClick={[Function]}
    type="button"
  >
    male
  </button>
  <button
    className="bg-btn bg-btn--inactive"
    data-value="female"
    key="female"
    onClick={[Function]}
    type="button"
  >
    female
  </button>
</div>
`;
